runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1325,6 +1325,12 @@ def DeleteCommands()
   sign_undefine(BreakpointSigns->map("'debugBreakpoint' .. v:val"))
 enddef
 
+def QuoteArg(x: string): string
+  var x_escaped = x ->substitute('\\', '\\\\', 'g')
+    ->substitute('"', '\\"', 'g')
+
+  return  x_escaped[0] == '"' && x_escaped[-1 : ] == '"' ? x_escaped : $'"{x_escaped}"'
```

Comment:
Is it possible for the first character to be a `"` and the last character to not be a `"`?

Reply:
That is what I was wondering also, but I preferred the approach "better safe than sorry".

Reply:
> I preferred the approach "better safe than sorry".

I'm not sure how turning one mal-formatted string into a different mal-formatted string is "safe".

Reply:
I'm confused (obvious to all at this point). Some comments about input assumptions? In any event, the first character will never be a `"` in the ternary test; it would have been converted to `\"` by the `substitute`.

First looking at this code I had thought the idea was to handle something already quoted; but I misunderstood.